app/Repositories/ProfileRepository.php

```diff
@@ -115,6 +115,29 @@ public function sortGroupsByDisplayOrder($grouped, $groups)
             ->toArray();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getProfilesByGroupOrderPiped($site_id, $groups)
+    {
+        $profile_listing = $this->getProfiles($site_id);
+
+        $group_order = explode('|', $groups);
```

Comment:
Why a pipe instead of a comma?

Reply:
This is being pulled in from custom fields, and the "standard" we've been using has been a pipe. It's done on the Law site (where the code came from) and there have been a few sites since that use a pipe to denote different profile_group_ids.